pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "read_version[toml] >= 0.3.0"]
+build-backend = "setuptools.build_meta"
+
+[tool.read_version]
+version = "pycodestyle:__version__"
```

Comment:
I'm -1 on adding pyproject.toml at this time since it changes how users need to install this

you can get the same thing in `setup.cfg` by using `attr: pycodestyle.__version__`

Reply:
>since it changes how users need to install this

`python3 ./setup.py bdist_wheel` should work fine, as `pip install .` do.

>you can get the same thing in setup.cfg by using attr: pycodestyle.__version__

I don't think so. We should never import the package being built from its setup script. It would either break or interfere to building having as cwd a dir other than git repo root.

Reply:
YAGNI also, you can do this directly with setup.cfg